web_m2x_options/static/src/components/form.esm.js

```diff
@@ -14,15 +14,31 @@ import {Many2OneField, many2OneField} from "@web/views/fields/many2one/many2one_
 import {PartnerMany2XAutocomplete} from "@partner_autocomplete/js/partner_autocomplete_many2one";
 import {ReferenceField} from "@web/views/fields/reference/reference_field";
 import {X2ManyField} from "@web/views/fields/x2many/x2many_field";
+import {Many2OneAvatarUserField} from "@mail/views/web/fields/many2one_avatar_user_field/many2one_avatar_user_field";
+import { SaleOrderLineProductField } from '@sale/js/sale_product_field';
```

Comment:
This add-on is not depended on odoo/sale. So this component may not exist at runtime.